packs/hpe-icsp/actions/icsp_server_attributes_get.yaml

```diff
@@ -0,0 +1,33 @@
+---
+  name: "icsp_server_attributes_get"
+  runner_type: "run-python"
+  description: "Retrieve Attributes set against Server"
+  enabled: true
+  entry_point: "icsp_server_attributes_get.py"
+  parameters:
+    mids:
+      type: "array"
+      description: "Server MID list, Comma seperated."
```

Comment:
I would personally remove "comma separated" part since this is a client (CLI in this case) implementation details.